com.ibm.wala.ide.tests/.settings/org.eclipse.jdt.core.prefs

```diff
@@ -84,8 +84,8 @@ org.eclipse.jdt.core.compiler.problem.redundantNullAnnotation=error
 org.eclipse.jdt.core.compiler.problem.redundantNullCheck=error
 org.eclipse.jdt.core.compiler.problem.redundantSpecificationOfTypeArguments=error
 org.eclipse.jdt.core.compiler.problem.redundantSuperinterface=error
-org.eclipse.jdt.core.compiler.problem.reportMethodCanBePotentiallyStatic=error
-org.eclipse.jdt.core.compiler.problem.reportMethodCanBeStatic=error
+org.eclipse.jdt.core.compiler.problem.reportMethodCanBePotentiallyStatic=ignore
```

Comment:
Without this change ECJ complains that a JUnit test method should be static

Reply:
If we want to receive these warnings for *most* of the code in this subproject, then it would be better to leave this Eclipse warning enabled and just suppress it for specific methods where we disagree with Eclipse's recommendation. Is that the case here? Or is this diagnostic wrong so often that it's impractical to suppress each wrong instance?

Reply:
I think it's a lint that should be disabled on test code if it's not aware of JUnit, as `@Test` methods should _not_ be static (AFAIK).  Plus the motivation for this lint, which is mostly performance I think, is moot for test code.  Unfortunately, we don't have a good way to enable / disable ECJ checks at the source set granularity, so I'm in favor of just disabling the check for the module (it is disabled in at least some other WALA modules).

Reply:
OK.